src/test/java/wromaciej/hvac_sim/thermo/fluids/IdealProcessTest.java

```diff
@@ -16,26 +16,26 @@
 import javax.measure.unit.NonSI;
 import javax.measure.unit.SI;
 
-import java.util.ArrayList;
-
 import static org.junit.Assert.assertEquals;
 
 public class IdealProcessTest {
 
-    static {
-        //load library
-        FluidData.loadLibrary();
-    }
+
 
     @Test
     public void shouldReturnSameTemperatureDifferenceByIsobaricHeatingOfWaterAndEquationCalculation(){
         //GIVEN
+        FluidData fluidData = new FluidData();
+        fluidData.loadLibrary();
+        FluidFactory fluidFactory= new FluidFactory(fluidData);
+        IdealProcess idealProcess = new IdealProcess(fluidFactory);
+
         Parameter<Temperature> temperatureBeforeIsobaricProcess = new Parameter<>(ParameterType.TEMPERATURE, SI.CELSIUS.asType(Temperature.class), 20);
         Parameter<Temperature> temperatureAfterIsobaricProcess = new Parameter<>(ParameterType.TEMPERATURE, SI.CELSIUS.asType(Temperature.class), 50);
```

Comment:
you could create helper method e.g.
```
private Parameter<Temperature> temperature(int degree) { ... }
```
then you would have here 
```
 Parameter<Temperature> temperatureBeforeIsobaricProcess = temperature(20);
```
all will be much shorter and without boilerplate. Similar things you could use in other places.